kakao/src/main/kotlin/io/github/kakaocup/kakao/common/matchers/SpanDrawableMatcher.kt

```diff
@@ -18,65 +18,76 @@ import io.github.kakaocup.kakao.common.utilities.getResourceDrawable
 import org.hamcrest.Description
 import org.hamcrest.TypeSafeMatcher
 
+/**
+ * Matches given drawable with specified span
+ *
+ * @param resId Drawable resource to be matched (default is -1)
+ * @param drawable Drawable instance to be matched (default is null)
+ * @param toBitmap Lambda with custom Drawable -> Bitmap converter (default is null)
+ * @param queryStart Index of the character that is the beginning of the range of text to which span is attached (default is 0)
+ * @param queryEnd Index of the character that is the end of the range of text to which span is attached (default is null)
+ * @param spanIndex Index of span in string's spans array (default is 0)
+ */
+@Suppress("LongParameterList")
 class SpanDrawableMatcher(
     @DrawableRes private val resId: Int = -1,
     private val drawable: Drawable? = null,
     @ColorRes private val tintColorId: Int? = null,
     private val toBitmap: ((drawable: Drawable) -> Bitmap)? = null,
     private val queryStart: Int = 0,
-    private var queryEnd: Int? = null,
+    private val queryEnd: Int? = null,
     private val spanIndex: Int = 0
 ) : TypeSafeMatcher<View>(View::class.java) {
 
     override fun describeTo(desc: Description) {
-        desc.appendText("with drawable id $resId")
+        desc.appendText("with span drawable id $resId or provided instance")
     }
 
     override fun matchesSafely(view: View?): Boolean {
-
-        if (view !is TextView && drawable == null) {
+        if (view !is TextView?) {
```

Comment:
At first, I took this comparison from DrawableMatcher
`view !is TextView && drawable == null`
but I can't understand why there was a comparison of drawable with null. So I removed it. Let me know if there is an error and a second comparison is needed here